sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Azure.AI.FormRecognizer.Tests.csproj

```diff
@@ -18,4 +18,8 @@
     <ProjectReference Include="$(AzureCoreTestFramework)" />
     <ProjectReference Include="..\src\Azure.AI.FormRecognizer.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="samples\V4\" />
+  </ItemGroup>
```

Comment:
I think we can skip adding this. My guess is that it was added because the folder is empty, but it should include the folder by default.

Reply:
Ohh I didn't notice that. And now that I think about it, it might be better to just delete the folder and have the current samples directly in the `Samples` folder.
That way we "make it more complicated" to find the V3 samples, and by default, we have the latest samples.

what do you think?

Reply:
Agreed. Makes more sense.

Reply:
ok. Removed